tests/test_init.py

```diff
@@ -22,44 +22,35 @@
 import subprocess
 import mock
 from mock import Mock
+from requests.exceptions import RequestException
 
 import skt
 
 
-class TestInit(unittest.TestCase):
-    """Test cases for skt's __init__.py"""
-
-    def test_stringify_with_integer(self):
-        """Ensure stringify() can handle an integer"""
-        myinteger = int(42)
-        result = skt.stringify(myinteger)
-        self.assertIsInstance(result, str)
-        self.assertEqual(result, str(myinteger))
-
-    def test_stringify_with_string(self):
-        """Ensure stringify() can handle a plain string"""
-        mystring = "Test text"
-        result = skt.stringify(mystring)
-        self.assertIsInstance(result, str)
-        self.assertEqual(result, mystring)
-
-    def test_stringify_with_unicode(self):
-        """Ensure stringify() can handle a unicode byte string"""
-        myunicode = unicode("Test text")
-        result = skt.stringify(myunicode)
-        self.assertIsInstance(result, str)
-        self.assertEqual(result, myunicode.encode('utf-8'))
-
-    def test_parse_bad_patchwork_url(self):
-        """Ensure parse_patchwork_url() handles a parsing exception"""
-        patchwork_url = "garbage"
-        with self.assertRaises(Exception) as context:
-            skt.parse_patchwork_url(patchwork_url)
-
-        self.assertTrue(
-            "Can't parse patchwork url: '{}'".format(patchwork_url)
-            in context.exception
-        )
+class TestIndependent(unittest.TestCase):
+    """Test cases for independent functions in __init__.py"""
+
+    def test_invalid_patch_url(self):
+        """Ensure get_patch_mbox() throws exception if the URL is invalid"""
+        self.assertRaises(RequestException,
+                          skt.get_patch_mbox,
+                          'this-is-invalid')
+
+    def test_nonexistent_patch_subject(self):
+        """Ensure get_patch_subject() handles nonexistent 'Subject' in mbox"""
+        mbox_body = 'nothing useful here'
+        self.assertEqual('subject', skt.get_patch_subject(mbox_body))
+
+    def test_ok_patch_subject(self):
+        """Ensure get_patch_subject() returns correct 'Subject' if present"""
+        mbox_body = 'From Test Thu May 2 17:49:51 2018\nSubject: GOOD SUBJECT'
+        self.assertEqual('GOOD SUBJECT', skt.get_patch_subject(mbox_body))
+
+    def test_encoded_patch_subject(self):
+        """Ensure get_patch_subject() correctly decodes 'Subject'"""
+        mbox_body = ('From Test Thu May 2 17:49:51 2018\n'
+                     'Subject: =?utf-8?q?=5BTEST=5D?=')
```

Comment:
Perhaps also test with that example from the RFC I mention in the other comment.